community/kernel/src/main/java/org/neo4j/kernel/impl/store/id/RenewableBatchIdSequences.java

```diff
@@ -58,9 +67,9 @@ public void close()
     {
         for ( StoreType type : StoreType.values() )
         {
-            if ( type.isRecordStore() )
+            if ( type.isRecordStore() && !type.isLimitedIdStore() )
             {
-                idGenerator( type ).close();
+                ((Resource)idGenerator( type )).close();
```

Comment:
Would it make sense to instead check if the respective IdGenerator implements Resource?